presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloSplitManager.java

```diff
@@ -57,7 +57,7 @@ public AccumuloSplitManager(
     }
 
     @Override
-    public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableLayoutHandle layout)
```

Comment:
I'm wondering whether this should be encoded in the layout, but I can't say without seeing how, exactly, this is derived. My intuition is that this is property of the plan shape, therefore, determined during planning/optimization (vs a runtime scheduling knob)